### ISSUE TITLE
Service Usage Matrix throws exception

### ISSUE THREAD

**Issue by cvanlith** [flamingo-geocms/flamingo#335]
HTTP Status 500 - ActionBean execution threw an exception.

type Exception report

message ActionBean execution threw an exception.

description The server encountered an internal error (ActionBean execution threw an exception.) that prevented it from fulfilling this request.

exception

net.sourceforge.stripes.exception.StripesServletException: ActionBean execution threw an exception.
	net.sourceforge.stripes.controller.DispatcherServlet.service(DispatcherServlet.java:183)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:722)
	net.sourceforge.stripes.controller.StripesFilter.doFilter(StripesFilter.java:247)
	nl.b3p.web.filter.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:122)
root cause

javax.xml.transform.TransformerException: The content of elements must consist of well-formed character data or markup.
	org.apache.xalan.transformer.TransformerImpl.fatalError(TransformerImpl.java:744)
	org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:720)
	org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:1192)
	org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:1170)
	nl.b3p.viewer.admin.stripes.ServiceUsageMatrixActionBean.transformXml(ServiceUsageMatrixActionBean.java:208)
	nl.b3p.viewer.admin.stripes.ServiceUsageMatrixActionBean.view(ServiceUsageMatrixActionBean.java:151)
	sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:606)
	net.sourceforge.stripes.controller.DispatcherHelper$6.intercept(DispatcherHelper.java:467)
	net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:158)
	org.stripesstuff.plugin.session.SessionStoreInterceptor.intercept(SessionStoreInterceptor.java:52)
	net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:155)
	org.stripesstuff.plugin.security.SecurityInterceptor.interceptEventHandling(SecurityInterceptor.java:196)
	org.stripesstuff.plugin.security.SecurityInterceptor.intercept(SecurityInterceptor.java:126)
	net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:155)
	net.sourceforge.stripes.controller.BeforeAfterMethodInterceptor.intercept(BeforeAfterMethodInterceptor.java:113)
	net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:155)
	org.stripesstuff.plugin.waitpage.WaitPageInterceptor.intercept(WaitPageInterceptor.java:175)
	net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:155)
	net.sourceforge.stripes.controller.ExecutionContext.wrap(ExecutionContext.java:74)
	net.sourceforge.stripes.controller.DispatcherHelper.invokeEventHandler(DispatcherHelper.java:465)
	net.sourceforge.stripes.controller.DispatcherServlet.invokeEventHandler(DispatcherServlet.java:278)
	net.sourceforge.stripes.controller.DispatcherServlet.service(DispatcherServlet.java:160)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:722)
	net.sourceforge.stripes.controller.StripesFilter.doFilter(StripesFilter.java:247)
	nl.b3p.web.filter.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:122)

**Comment by mprins** [comment]
Er wordt xml gegenereerd als:

```xml

<application>
	<id>6</id>

	<services>
		<2>
			<styleLibraries></styleLibraries>
			<id>2</id>
			<exception_type>application/vnd.ogc.se_inimage</exception_type>
			<useProxy>false</useProxy>
			<protocol>wms</protocol>

```
wat geen geldige xml is, (xml elementen mogen niet met een cijfer beginnen), kennelijk gaat er iets niet goed met de JSON -> xml omzetting in de stap ervoor waar de `services` nodes worden aangemaakt

zie: https://github.com/flamingo-geocms/flamingo/blob/master/viewer-admin/src/main/java/nl/b3p/viewer/admin/stripes/ServiceUsageMatrixActionBean.java#L149

**Comment by mprins** [comment]
closed by 66d7acb45353b10b84aaf930f40e77a7fd249987
